bip-0360.mediawiki

```diff
@@ -0,0 +1,607 @@
+<pre>
+  BIP: 360
+  Title: QuBit: SegWit v3 spending rules (P2QRH)
```

Comment:
After having just read the BIP again, I’m not sure why the term "QuBit" appears in the title. Wouldn’t the main topic simply be:

```suggestion
  Title: Pay to Quantum Resistant Hash
```